docker/Dockerfile.download

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu
+ADD https://github.com/yegor256/aibolit/releases/download/v1.0.0/dataset.zip .
+ADD https://github.com/yegor256/aibolit/releases/download/v1.0.0/halstead.jar .
```

Comment:
@acheshkov if you do this:

```
RUN wget https://github.com/yegor256/aibolit/releases/download/v1.0.0/halstead.jar
ADD halstead.jar
```

The problem will be solved, I believe.

Reply:
Thanks, done